latex/main.tex

```diff
@@ -38,21 +55,76 @@ \section{The website and how it works}
 ~\\
 
 \subsection{Login and the home page}
-The login page is simple, use your username and password to login to the website. This will redirect you to the homepage. On the top of this website there is a navigation bar, which is used to visit the several available pages.
+This section will tell you about the basics: login, logout and the homepage. Here the instruction for logging in can be found. We'll start with the login.
+~\\
+
+\subsubsection{Login and Logout}
+The login page is simple, use your username and password to login to the website.
+
+\begin{enumerate}
+\item Use the upper input field for your username.
+\item Use the lower one for your password.
+\item Finally, click the "login" button.
+\end{enumerate}
+
+\noindent
+\includegraphics[width=\textwidth]{login}
+
+\noindent
+After completing these steps you are being redirected to the homepage.
+\\
+On the top of this website there is a navigation bar, which is used to visit the several available pages. These pages include: the home page the map page and the admin page. The latter is only shown if the current user is an admin.
+\\
+If you want to log out, simply click the logout button at the very end of the navigation bar.
 \\
-\indent
-(Wat er dan ook op de hoofdpagina moet staan.) blablablablablablablabla bla bla.
+Both the login page and home page contain buttons to switch between the languages Dutch and English.
+
+\pagebreak
+
+\subsubsection{The homepage}
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
Gebruik de lipsum-package, zie [LaTeX: Generate dummy text](http://blog.chapagain.com.np/latex-generate-dummy-text-lorem-ipsum-in-your-document/).